README.md

```diff
@@ -55,3 +55,9 @@ TODO non-functional:
 - document the release process, testing, debugging in the README
 - document how to run examples
 - acceptance tests
+
+
+### Documentation
```

Comment:
I think that it is sufficent to add _generate documentation_ entry to _development steps_ above (instead of introducting new sub-chapter)

Reply:
Done